File: future-engine/src/main/java/com/sotatek/future/exception/ExceedRiskLimitException.java
package com.sotatek.future.exception;

import com.sotatek.future.util.MarginBigDecimal;

public class ExceedRiskLimitException extends MarginException {

  public ExceedRiskLimitException(MarginBigDecimal riskValue) {
    super("Exceed risk limit: " + riskValue.toString());
  }
}

File: spot-backend/app/Exceptions/ExceedRiskLimitException.php
<?php

namespace App\Exceptions;

use App\Exceptions\MarginException;
use App\Consts;

class ExceedRiskLimitException extends MarginException
{
    public $riskValue;

    public function __construct($riskValue)
    {
        $message = 'Exceed risk limit';
        $this->riskValue = $riskValue;
        parent::__construct($message);
    }
}
